synapse/lib/types.py

```diff
@@ -1967,11 +1967,68 @@ def cmpr(valu):
 
         return cmpr
 
+    def _ctorCmprLt(self, text):
+
+        if isinstance(text, str):
+            strip = text.strip()
+            if strip.endswith('*'):
+                tick, tock = s_time.wildrange(strip[:-1])
+                def cmpr(valu):
+                    return valu < tock
+                return cmpr
+
+        return IntBase._ctorCmprLt(self, text)
+
+    def _ctorCmprLe(self, text):
+
+        if isinstance(text, str):
+            strip = text.strip()
+            if strip.endswith('*'):
+                tick, tock = s_time.wildrange(strip[:-1])
+                def cmpr(valu):
+                    return valu <= tock
+                return cmpr
+
+        return IntBase._ctorCmprLe(self, text)
+
     def _ctorCmprEq(self, text):
 
+        if isinstance(text, str):
+            strip = text.strip()
+            if strip.endswith('*'):
+                tick, tock = s_time.wildrange(strip[:-1])
```

Comment:
Should this (and other uses of `[:-1]` be `rstrip('*')` ? The only way the wildparse works is that the _raw_parse ends up dropping the non numeric characters.

Reply:
We have confirmed already ( using `endswith()` ) that the last character is the one we're looking for and i'd rather explicitly remove just that one.